test/e2e_node/pod_resize_test.go

```diff
@@ -1131,6 +1131,218 @@ func doPodResizeTests() {
 				},
 			},
 		},
+		{
+			name: "Burstable QoS pod, one container with only cpu requests - increase cpu requests",
```

Comment:
I refactored pod resize e2e tests in https://github.com/kubernetes/kubernetes/pull/128143. Is it possible to update the test file?